call_EtherscanAPI.jl

```diff
@@ -1,33 +1,55 @@
 using HTTP, JSON
 
+# to override with your Etherscan API key
+API_token = ""
+# first block to start from
+first_block = 6290000
+log_result_topic = "0x8dd0b145385d04711e29558ceab40b456976a2b9a7d648cc1bcd416161bf97b9"
+contract_address = "0xA52e014B3f5Cc48287c2D483A3E026C32cc76E6d"
```

Comment:
As it may change on contract update, it's now exposed as a variable to ease maintaining the script

Reply:
This makes good sense.